conductor/resources/submitter.ui

```diff
@@ -527,48 +562,78 @@
             </widget>
            </item>
            <item>
-            <widget class="QCheckBox" name="ui_preemptible_chkbx">
-             <property name="layoutDirection">
-              <enum>Qt::RightToLeft</enum>
-             </property>
-             <property name="text">
-              <string>Preemptible</string>
-             </property>
-             <property name="isFileUserPref" stdset="0">
-              <bool>true</bool>
-             </property>
-            </widget>
-           </item>
-           <item>
-            <widget class="QWidget" name="ui_resource_wgt" native="true">
+            <widget class="QWidget" name="ui_preemptible_wgt" native="true">
              <property name="toolTip">
-              <string>[optional] Specify a resource string to associate the Conductor job with</string>
+              <string>A preemptible instance is up to 60% less expensive, but may be terminated before it completes a task.
+Best to use when not performing critical work.</string>
```

Comment:
Can we reword this to say:  "A preemptible instance can be significantly less expensive, but may be terminated before it completes a task."

Reply:
ha, I was just asking mac about this. Sure I can do that.  Figured vaguely defined savings wouldn't be terribly inspiring (60% means something to people.  10% maybe not so much.  Yet both could be described as "immense savings" )

Reply:
addressed